### PR TITLE
Qsw 696 - Progress

### PR DESCRIPTION
**Pull request by claudio525**
- Adds the progress script.
- Changed cybershake estimation to use constants.
- The diff shows dashboard.py as a new file,  but this was already merged in PR https://github.com/ucgmsim/slurm_gm_workflow/pull/60 , so I think thats just a github issue....

Requires PR https://github.com/ucgmsim/qcore/pull/72

### CODE REVIEW

File: dashboard/cybershake_progress.py [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+    with open(cybershake_list, "r") as f:
+        lines = f.readlines()
+
+    fault_names, r_counts = [], []
+    for ix, line in enumerate(lines):
+        line_l = line.split(" ")
+        fault_names.append(line_l[0].strip())
+
+        try:
+            r_counts.append(int(line_l[1].strip().rstrip("r")))
```
Comment (jasonmotha) [inline dashboard/cybershake_progress.py]: We need to standardise the 'r'

File: dashboard/cybershake_progress.py [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+                    progress_df.loc[fault, ("total", COMPLETED_R_COUNT_COL)],
+                    progress_df.loc[fault, ("total", R_COUNT_COL)],
+                ),
+                "True"
+                if progress_df.loc[fault, ("total", MISSING_DATA_FLAG_COL)]
+                else "False",
+            )
+        )
+
+    lf_act = progress_df[const.ProcessType.EMOD3D.str_value, ACT_CORE_HOURS_COL].sum()
```
Comment (jasonmotha) [inline dashboard/cybershake_progress.py]: It almost seems nicer if we loop it. But that has disadvantages too.

File: dashboard/cybershake_progress.py (hunk 1 above)
Comment (jasonmotha) [reply dashboard/cybershake_progress.py]: https://github.com/ucgmsim/slurm_gm_workflow/blob/master/scripts/cybershake/install_cybershake.sh#L28

NKR are all currently valid realisation specifications. A conversation about wether we are deprecating the old ones is a worthwhile.

File: dashboard/cybershake_progress.py (hunk 2 above)
Comment (claudio525) [reply dashboard/cybershake_progress.py]: Good catch, no idea why I did it that way, its rather ugly....

File: dashboard/cybershake_progress.py (hunk 1 above)
Comment (claudio525) [reply dashboard/cybershake_progress.py]: Have added "n" and "k"

**Pull request merged**